workflows/bizreach/check_progress.py

```
@@ -0,0 +1,91 @@
+    manager = ProgressBrowser()
+    
+    def signal_handler(signum, frame):
+        print('\n🛑 終了シグナルを受信しました...')
+        asyncio.create_task(manager.stop())
+        sys.exit(0)
+    
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+    
+    try:
+        page = await manager.start()
```

Flake8 (7.2.0)

[error] local variable 'page' is assigned to but never used

(F841)